### PR TITLE
Slate, typedoc, raiden

### PR DESCRIPTION
**Pull request by kilrau**

### CODE REVIEW

File: Developer-Guide.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ Consider using the following plugins for working with `xud`.
 
 Adding the two files from [this gist](https://gist.github.com/sangaman/117af412eefc28c4f763c0152ddd3b99) into a `.vscode` folder within the folder where you've cloned `xud` will automatically provide debug configurations and general settings that are helpful when developing `xud`. 
 
+## Documentation
+
+- [TypeDoc Code Documentation](http://typedoc.exchangeunion.com/)
+- [Slate API Documentation](http://api.exchangeunion.com)
```
Comment (erkarl) [inline Developer-Guide.md]: Maybe just call it API documentation?

File: Exchange-Guide.md [hunk 2]
```diff
@@ -31,6 +31,15 @@ Default is `matching` mode, which means that `xud` acts as order book & matching
...
+* Restarting or stopping `xud` clears all orders from its order book. When `xud` boots up, it automatically reconnects to its peers and fetches all open orders.
+* `xud` relies on is its own (local) view of orders. This is also how it determines if it acts as a maker or a taker when a match with a peer order is found. If the peer order reached the order book first it acts as taker, if the local order reached the order book first it acts as maker. The swap protocol defines that only as taker, `xud` actively initiates a swap, as maker it'll wait for the swap to be initiated by the remote taker. For the same reason, `xud` never matches incoming peer orders with the existing order book; only incoming local orders. Due to network latency it can now happen that both parties determine themselves to be the maker when they place their orders nearly simultaneously. In this case, a swap won't be initiated - nothing happens.
+* To query existing orders one can use `ListOrders`, in addition to the `existing` flag for the `SubscribeAddedOrders` streaming call.
```
Comment (erkarl) [inline Exchange-Guide.md]: /s/SubscribeAddedOrders/SubscribeOrders

Comment (kilrau) [reply Exchange-Guide.md]: fixed for all

File: Developer-Guide.md (hunk 1 above)
Comment (kilrau) [reply Developer-Guide.md]: Done

**Comment by kilrau** [comment]
added raiden stuff to simnet guide, addressed your review @erkarl

**Review by ghost — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
LGTM. Needs a rebase.

**Pull request merged**